src/test/isolation2/sql/copy_interrupt.sql

```diff
@@ -0,0 +1,27 @@
+!\retcode gpconfig -c client_connection_check_interval -v 20s;
+!\retcode gpstop -u;
+
+1: SELECT gp_inject_fault('proc_kill', 'suspend', dbid)
+    FROM gp_segment_configuration 
+    WHERE role = 'p' AND content = -1;
+
+0: CREATE TABLE copy_interrupt_table(a int);
+0&: COPY copy_interrupt_table FROM PROGRAM 'while true; do echo 1; sleep 1; done | cat -';
+
+0t:
+
+1: SELECT gp_wait_until_triggered_fault('proc_kill', 1, dbid)
+    FROM gp_segment_configuration
+    WHERE role = 'p' AND content = -1;
+1: SELECT gp_inject_fault('proc_kill', 'resume', dbid)
+    FROM gp_segment_configuration 
+    WHERE role = 'p' AND content = -1;
+-- There shouldn't be any backend to terminate by this point
+-- Still, terminate it to not leave hanging processes even in case test fails
+-- For the test to pass, query should return zero rows
+SELECT pg_terminate_backend(pid)
+    FROM pg_stat_activity where query LIKE 'COPY copy_interrupt_table FROM PROGRAM%';
```

Comment:
it is not work with current fault injector approach
I suggest remove these lines

Reply:
It is still possible for the test to fail even with fault injectors (there is a 10 minute timeout in case the backend isn't killed). In that case, this line does actually prevent the backend from hanging indefinitely if test fails for some reason.